src/test/java/seedu/address/storage/JsonInternBookStorageTest.java

```diff
@@ -63,7 +63,7 @@ public void readInternBook_invalidAndValidCompanyInternBook_throwDataLoadingExce
     @Test
     public void readAndSaveInternBook_allInOrder_success() throws Exception {
         Path filePath = testFolder.resolve("TempAddressBook.json");
```

Comment:
maybe the name of the TempAddressBook.json could be changed to TempInternBook.json but you would need to probably change other instances of this as well, and i am unsure are we allowed to change the json file, maybe we can ask the group